### ISSUE TITLE
[CP] Slider in Drawer Doesn't Slide 

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#103662]
### issue_link

https://github.com/flutter/flutter/issues/103566

### Commit Hash

4960492

### Target

stable

### Impacted Users

Flutter Developers that ship on android using Drawer+Slider

### Impact Description

When nested a horizontal slider in a any widget that has a horizontal drag gesture, such as a PageView or Drawer, the slider drag does not functiona and instead only taps are registered.

### Workaround

The workaround is to wrap the application in a MediaQuery that overrides that touchSlop to framework default value of 16:

```dart
MaterialApp(
 home: MediaQuery(
   data: MediaQueryData.fromWindow(window).copyWith(gestureSettings: DeviceGestureSettings(touchSlop: kTouchSlop)))
   child: ...
 )

```

### Risk

low

### Test Coverage

yes

### Validation Steps

The issue https://github.com/flutter/flutter/issues/103566 describes a scenario that does not work on many physical android devices where the platform touch slop is less than the configured framework touch slop. The unit test will also cover this case.

**Comment by itsjustkevin** [comment]
@HansMuller

**Comment by jonahwilliams** [comment]
@goderbauer

**Comment by goderbauer** [comment]
@itsjustkevin I approve of this cherry-pick.

**Comment by LostInDarkMath** [comment]
When will this fix be released? I tried the new beta version `3.1.0` but the issue is still there.

**Comment by jonahwilliams** [comment]
We don't cherry pick fixes to betas usually, it will be in the next stable hotifx

**Comment by itsjustkevin** [comment]
Flutter 3.0.2

**Comment by LostInDarkMath** [comment]
Thank you, it works!

**Comment by LostInDarkMath** [comment]
But it only works fors `Slider` and not for `RangeSlider` as you can see [here](https://github.com/flutter/flutter/issues/105833)

Is there a chance to see a fix in Flutter 3.0.3?
